lib/jxl.cmake

```diff
@@ -402,6 +402,9 @@ else ()
   )
 endif ()
 
+# Generate version.h
+configure_file("jxl/version.h.in" "include/jxl/version.h")
```

Comment:
Might want to make sure proper source and build dirs are used, i.e. ${CMAKE_CURRENT_SOURCE_DIR}/jxl/version.h.in and ${CMAKE_CURRENT_BINARY_DIR}/include/jxl/version.h

Reply:
from the docs ( https://cmake.org/cmake/help/latest/command/configure_file.html?highlight=relative%20path ) those are the default values. I don't have a preference as to whether they should be explicitly defined or not.